Sources/Localization/en.lproj/Localizable.strings

```diff
@@ -304,6 +304,7 @@
 "call-to-action.summary" = "Take a photo of the front";
 "call-to-action.ingredients" = "Take a photo of the ingredients";
 "call-to-action.nutrition" = "Take a photo of the nutrition facts";
+"call-to-action.edit" = "Please edit this product.";
```

Comment:
I would personalize to get a clear view of the expected result.
Please do XYZ to get ABC

Reply:
"Edit product details to have it analysed"?

Reply:
"Complete the product to get it scored" ?